### PR TITLE
proposed V8 changes

### PR DESCRIPTION
**Pull request by ramosaj**
* Populate code for all tables
* some nomenclature changes 
* Trial indices are now sequential and consistent, representing the convention:
    * end frame f0 indicates trial ended closer to f1 than f0 (rounded frame -1)
    * start frame of f1 indicates trial started closer to f1 (rounded frame)

### CODE REVIEW

File: phase3/func.py [hunk 1]
```diff
@@ -309,4 +324,199 @@ def fetch_oracle_raster(unit_key):
     oracle_traces -= np.min(oracle_traces,axis=(1,2),keepdims=True) # normalize the oracle traces
     oracle_traces /= np.max(oracle_traces,axis=(1,2),keepdims=True) # normalize the oracle traces
     oracle_score = (nda.Oracle & unit_key).fetch1('pearson') # fetch oracle score
-    return oracle_traces, oracle_score
\ No newline at end of file
+    return oracle_traces, oracle_score
+
+def get_timing_offset(key):
```
Comment (ramosaj) [inline phase3/func.py:329]: function description

**Comment by ramosaj** [comment]
* Need to add Docstrings to all the functions in func.py 
* Make note about 4-9

File: phase3/nda.py [hunk 2]
```diff
@@ -215,9 +397,59 @@ class Trial(dj.Manual):
...
+                
+                start_index = (field1_pixel0 < trial_flip_times[0]).sum() - 1
+                end_index = (field1_pixel0 < trial_flip_times[-1]).sum() - 1
+                start_time = trial_flip_times[0]
+                end_time = trial_flip_times[-1]
+                if(idx > 0):
+                    if(data[idx-1]['end_idx'] == start_index):
+                        print('ding')
+                        t0 = data[idx-1]
+                        med = (start_time  +t0['end_frame_time'])/2
```
Comment (ramosaj) [inline phase3/nda.py]: should be 
``` med = start_time ```

**Comment by paulgfahey** [comment]
These changes have been wrapped into a forthcoming pull request

**Pull request closed without merging**